dask/array/chunk_types.py

```diff
@@ -0,0 +1,60 @@
+from numbers import Number
+
+import numpy as np
+
+
+# Start list of valid chunk types, to be added to with guarded imports
+_HANDLED_CHUNK_TYPES = [np.ndarray, np.ma.MaskedArray]
+
+
+def register_chunk_type(type):
+    """ Register the given type as a valid chunk and downcast array type"""
```

Comment:
Since this is exposed as part of the top-level `dask.array` namespace, would you mind giving this a longer docstring? Would be good to at least better describe what you mean by "valid chunk and downcast array type", and maybe give an example of when you'd use this method.